.travis.yml

```diff
@@ -7,6 +7,6 @@ script: ./run.sh
 deploy:
   skip_cleanup: true
   provider: script
-  script: ./configs/travis/deploy_github_pages.sh
+  script: . ./configs/travis.sh && . ./configs/deploy.sh
```

Comment:
is the extra `.` necessary?

Reply:
The `.` s were added to source the files. I didn't add them initially and seemed that the variables assigned in `travis.sh` were not usable in `deploy.sh`. But I'm not very sure whether this is the best way to do this since I'm not very familiar with shell scripts.

Reply:
ahhh I see. Same here. Let's wait for the senior developers to respond.

Reply:
That's one way to do it.

Alternatively, if you use `export` or `set` in the scripts i.e. `export var=value` then you should be able to do this `./configs/travis.sh && ./configs/deploy.sh` and still access the variables stored in the `env`.